src/code/z_lights.c

```diff
@@ -257,15 +257,15 @@ void LightContext_Init(PlayState* play, LightContext* lightCtx) {
 }
 
 void LightContext_SetAmbientColor(LightContext* lightCtx, u8 r, u8 g, u8 b) {
-    lightCtx->ambient.r = r;
-    lightCtx->ambient.g = g;
-    lightCtx->ambient.b = b;
+    lightCtx->ambientColor[0] = r;
+    lightCtx->ambientColor[1] = g;
+    lightCtx->ambientColor[2] = b;
 }
 
 void LightContext_SetFog(LightContext* lightCtx, u8 r, u8 g, u8 b, s16 near, s16 far) {
-    lightCtx->fogColor.r = r;
-    lightCtx->fogColor.g = g;
-    lightCtx->fogColor.b = b;
+    lightCtx->fogColor[0] = r;
+    lightCtx->fogColor[1] = g;
+    lightCtx->fogColor[2] = b;
```

Comment:
The colors have reverted to array from rgba?

Reply:
the environment dev loves using arrays for colors, and it was needed to match in some cases (in oot and probably here)

Reply:
Can confirm. At least one function iterates over them in a for loop. Just not in this PR.